tests/test_core/test_typesbase.py

```diff
@@ -43,16 +46,15 @@ def test_char_array_set_and_get(self):
             self.assertEqual(o[i], content[start:finish])
 
     def test_char_array_set_and_ffi_string(self):
-        ffi = cffi.FFI()
-        o = typesbase.CFFICDataWrapper("char [256]")
+        o = typesbase.CFFICDataWrapper("char [256]", _ffi)
         content = b"0123456789"
         content_len = len(content)
         for i in range(content_len):
             # Have Python 3 return a len()==1 bytes object
             o[i] = content[i:i+1]
         # must use wrapped cdata to apply ffi.string in this case
         # won't be needed for struct members as other tests verify
-        self.assertEqual(ffi.string(o._cdata), content)
+        self.assertEqual(_ffi.string(o._cdata), content)
```

Comment:
Should use dist.load() to retrieve an instance of ffi like we do in normal code.